config/locales/hyrax.de.yml

```diff
@@ -679,9 +679,9 @@ de:
         your_collections: Ihre Sammlungen
         your_works: Deine Werke
       nest_collections_form:
-        create_under: "'%{child_title}' wurde zu '%{parent_title}' hinzugefügt."
-        create_within: "'%{child_title}' wurde zu '%{parent_title}' hinzugefügt."
-        removed_relationship: "'%{child_title}' wurde aus '%{parent_title}' entfernt."
+        create_under: ''%{child_title}' wurde zu '%{parent_title}' hinzugefügt.'
```

Comment:
items with single quotes need to be wrapped in double quotes

Reply:
Just to make sure, line 682 should look like this "'%{child_title}' wurde zu '%{parent_title}' hinzugefügt." ?

Reply:
Yes Erik, that is correct, thank you for double checking!

Reply:
Perfect! Thank you, I'll get the rest of these fixed and will let you both know when it's ready for review.